include/game.h

```diff
@@ -475,6 +475,12 @@ void Game_NetworkInit(nrf24_t * nrf_ptr);
 #endif
 
 void Game_DrawDiamond(char c, uint8_t x, uint8_t y, uint8_t w, uint8_t h);
+//@param x: x coordinate of top corner of invisible square that bounds the diamond
```

Comment:
doxygen documentation blocks should start with /** and go before the thing being documented.